src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
```

Comment:
I recommend changing the variable name for consistency
```suggestion
    ? people.filter((person)=> person.sex === 'f'
```